src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -425,6 +428,13 @@ public PrivilegesEvaluatorResponse evaluate(final User user, String action0, fin
                         presponse.allowed = true;
                         return presponse;
                     }
+
+                    if(request instanceof IndicesRequest.Replaceable) {
```

Comment:
All other requests  under `if` are also `Replaceable`.

Reply:
made use of `Replacable` as much as possible